presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestSimplifyExpressions.java

```diff
@@ -114,13 +130,24 @@ public void testExtractCommonPredicates()
     }
 
     private static void assertSimplifies(String expression, String expected)
+    {
+        assertSimplifies(expression, expected, null);
+    }
+
+    private static void assertSimplifies(String expression, String expected, String rowExpressionExpected)
     {
         Expression actualExpression = rewriteIdentifiersToSymbolReferences(SQL_PARSER.createExpression(expression));
         Expression expectedExpression = rewriteIdentifiersToSymbolReferences(SQL_PARSER.createExpression(expected));
         Expression rewritten = rewrite(actualExpression, TEST_SESSION, new PlanVariableAllocator(booleanVariablesFor(actualExpression)), METADATA, LITERAL_ENCODER, SQL_PARSER);
         assertEquals(
                 normalize(rewritten),
                 normalize(expectedExpression));
+        TestingRowExpressionTranslator translator = new TestingRowExpressionTranslator(METADATA);
+        RowExpression actualRowExpression = translator.translate(actualExpression, TypeProvider.viewOf(TYPES));
+        RowExpression simplifiedByRowExpression = SimplifyRowExpressions.rewrite(actualRowExpression, METADATA, TEST_SESSION.toConnectorSession());
```

Comment:
I don't think `By` actually mean something.... Remove it from the variable name. Same for all other `By`s.